modules/gitops-logging-into-keycloak.adoc

```diff
@@ -0,0 +1,66 @@
+[id="logging-into-keycloak_{context}"]
+= Logging into Keycloak
+
+Log in to the Keycloak console after configuring the Argo CD instance to manage identities or roles and access provided to the various roles. To log in, first you have to get the Keycloak admin username and password.
+
+.Procedure
+
+. Get the Keycloak Route URL for Login:
```

Comment:
Check the capitalization here. Is `Route URL for Login` referring to a UI element? Or a value in a file somewhere?